hazelcast-jet-distribution/src/bin-regular/common.sh

```diff
@@ -12,19 +12,36 @@ if [ -z "$JAVA" ]; then
     exit 1
 fi
 
-CLASSPATH="$JET_HOME/lib/*:$CLASSPATH:$CLASSPATH_DEFAULT"
+# 1 -> Java 8 or earlier (1.8..)
+# 9, 10, 11 -> JDK9, JDK10, JDK11 etc.
+JAVA_VERSION=$(${JAVA} -version 2>&1 | sed -En 's/.* version "([0-9]+).*$/\1/p')
```

Comment:
Will this work for other JVM impls? Ibm, Azul...

Reply:
I tested on OpenJDK and HotSpot. These will need additional testing.

Reply:
The output of `java -version` is specified: https://www.oracle.com/technetwork/java/javase/versioning-naming-139433.html